### DISCUSSION TITLE
Why can't I move list items between different lists?

### DISCUSSION THREAD

**Discussion by spiderand707** [vslinko/obsidian-outliner#149] [Q&A] [answered]
Just updated to 1.2.1 and moving a list up and down in an outline with the default (or customised) hotkeys has stopped working. With the default ones it now selects all content above or below the insertion point.
Is this just me or is it brokenededed? There is no clash with those hotkeys in Obsidian.

**Comment by vslinko** [comment, marked as answer]
@spiderand707 Could you provide an example of a document that has a problem?

**Reply by spiderand707** [reply]
I think I've misunderstood the function so apologies. I thought you could move a full section including header like 'Second selection test' in the file below above say 'First selection test' with all of it's children at once but that doesn't work. I can only move list and sublist items in the file below. My misunderstanding I think?

[Test Note.md](https://github.com/vslinko/obsidian-outliner/files/6545311/Test.Note.md)

**Reply by vslinko** [reply]
@spiderand707 Yes, moving is not supported for lists. Please subscribe to #53 to be notified when it will be released.

**Reply by spiderand707** [reply]
ah cool, thanks, will do. Thanks for a great plug-in btw :)
